packages/gen-ai-hub/src/orchestration/orchestration-response.ts

```diff
@@ -0,0 +1,66 @@
+import { HttpResponse } from '@sap-cloud-sdk/http-client';
+import { createLogger } from '@sap-cloud-sdk/util';
+import {
+  CompletionPostResponse,
+  TokenUsage
+} from './client/api/schema/index.js';
+
+const logger = createLogger({
+  package: 'gen-ai-hub',
+  messageContext: 'orchestration-response'
+});
+/**
+ * Respresentation of Orchestration response.
+ */
+export class OrchestrationResponse {
+  /**
+   * Get the typed Completion Post Response.
+   */
+  public readonly data: CompletionPostResponse;
+  constructor(public readonly rawResponse: HttpResponse) {
+    this.data = rawResponse.data;
+  }
+
+  /**
+   * Usage of tokens in the response.
```

Comment:
```suggestion
   * Usage of tokens for the response.
```

Reply:
Not sure if for is the right suggestion, but in my head the tokens were used to create the response, so this might be more appropriate?

Reply:
I just copied this from the spec